graylog2-web-interface/src/views/stores/ViewStore.ts

```diff
@@ -156,15 +166,14 @@ export const ViewStore: ViewStoreType = singletonStore(
 
       return promise;
     },
-    load(view: View, isNew = false): Promise<ViewStoreState> {
+    load(view: View, isNew = false, queryId: string = undefined): Promise<ViewStoreState> {
       this.view = view;
       this.dirty = false;
 
-      /* Select selected query (activeQuery) or first query in view (for now).
+      /* Select query id passed thorugh URL, selected query (activeQuery) or first query in view (for now).
          Selected query might become a property on the view later. */
       const queries = view.state.keySeq().toList();
-      const firstQueryId = view?.search?.queries?.first()?.id ?? queries.first();
```

Comment:
Previously the default `activeQuery` was based on `view?.search?.queries?.first()?.id` to ensure we are always  defining the first search query as the active one: https://github.com/Graylog2/graylog2-server/pull/10790

Reply:
Thanks, good catch! Fixed it.